bot/seasons/evergreen/bookmark.py

```diff
@@ -45,19 +49,54 @@ def __init__(self, bot: commands.Bot):
         embed.set_author(name=target_message.author, icon_url=target_message.author.avatar_url)
         embed.set_thumbnail(url=bookmark_icon_url)
 
+        error_embed = discord.Embed(
+            title=random.choice(ERROR_REPLIES),
+            description=f"Please enable your DMs to receive the bookmark.",
+            colour=Colours.soft_red
+        )
+
         try:
             await ctx.author.send(embed=embed)
         except discord.Forbidden:
-            error_embed = discord.Embed(
-                title=random.choice(ERROR_REPLIES),
-                description=f"{ctx.author.mention}, please enable your DMs to receive the bookmark",
-                colour=Colours.soft_red
-            )
             await ctx.send(embed=error_embed)

```

Comment:
This should be self-deleting as well to avoid clogging the channel. Maybe change this to a normal message as well so the bot can mention the `ctx.author`
This can be done in a similar manner in line 95.

Reply:
It is been talked about and the decision was made to follow the theme of error message.

Reply:
Forgot to mention . added self deletion after 7.5 seconds. same as in error_handler